### PR TITLE
opera: 60.0.3255.170 -> 62.0.3331.43

### PR DESCRIPTION
**Pull request by xrelkd**
###### Motivation for this change

Update `opera` to the latest release.
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
@GrahamcOfBorg build opera

### CODE REVIEW

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 1]
```diff
@@ -92,11 +94,12 @@ let
 
 in stdenv.mkDerivation {
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]:
```suggestion
in stdenv.mkDerivation rec {
```

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 2]
```diff
@@ -92,11 +94,12 @@ let
...
 
-  name = "opera-${version}";
+  inherit pname;
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]:
```suggestion
```

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 3]
```diff
@@ -92,11 +94,12 @@ let
...
+  inherit version;
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]:
```suggestion
```

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 4]
```diff
@@ -40,9 +41,10 @@
 let
 
   mirror = https://get.geo.opera.com/pub/opera/desktop;
-  version = "60.0.3255.170";
+  pname = "opera";
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]:
```suggestion
```

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 5]
```diff
@@ -40,9 +41,10 @@
...
+  version = "62.0.3331.43";
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]:
```suggestion
```

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 6]
```diff
@@ -92,11 +94,12 @@ let
 
 in stdenv.mkDerivation {
 
-  name = "opera-${version}";
+  inherit pname;
+  inherit version;
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]:
```suggestion
  pname = "opera";
  version = "62.0.3331.43";

```

File: pkgs/applications/networking/browsers/opera/default.nix [hunk 7]
```diff
@@ -118,10 +121,10 @@ in stdenv.mkDerivation {
...
   '';
 
-  meta = {
-    homepage = https://www.opera.com;
+  meta = with lib; {
+    homepage = "https://www.opera.com";
     description = "Web browser";
-    platforms = [ "x86_64-linux" ];
-    license = stdenv.lib.licenses.unfree;
+    platforms = platforms.linux;
```
Comment (worldofpeace) [inline pkgs/applications/networking/browsers/opera/default.nix]: Please revert the change here. The binary is only for that platform.

**Review by worldofpeace — CHANGES_REQUESTED** [review]

**Comment by xrelkd** [comment]
@worldofpeace Thanks for the review!
@GrahamcOfBorg build opera

**Review by worldofpeace — APPROVED** [review]
Builds and executes for me locally on NixOS.
(can't be built by ofborg because unfree)
